### PR TITLE
Bug 1223346 - [TV] FTE for Marketplace

### PR DESCRIPTION
**Pull request by josyeh**
Merge Fischer's previous work of [FTE for marketplace](https://github.com/Fischer-L/fte-marketplace) into current codebase. Make some changes to make it suitable with marketplace website.

See the [visual spec](http://bit.ly/1lr4WNu) for more details.

### CODE REVIEW

File: src/dev.html [hunk 1]
```diff
@@ -21,7 +21,11 @@
     <script>
       (function(navigator) {
         if (!navigator.onLine) {
-          location.href = "offline.html";
+          location.href = 'offline.html';
```
Comment (spasovski) [inline src/dev.html:24]: thanks for keeping it consistent

File: src/tutorial/css/index.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+
+    background-color: #000;
+}
+
+#tutorial-container {
```
Comment (spasovski) [inline src/tutorial/css/index.css]: not a fan of id selectors since they have high specificity and are tricky to override

**Comment by spasovski** [comment]
seems good r+

File: src/tutorial/css/index.css (hunk 2 above)
Comment (josyeh) [inline src/tutorial/css/index.css]: Agree, will use class name instead.

**Pull request merged**